modules/authentication/proc-enabling-user-authentication-with-microsoft-azure.adoc

```diff
@@ -0,0 +1,485 @@
+
+Optional: Consider adding following optional fields:
+
+`domainHint`::
+Optional for single-tenant applications.
+You can reduce login friction for users with accounts in multiple tenants by automatically filtering out accounts from other tenants.
+If you want to use this parameter for a single-tenant application, uncomment and enter the tenant ID.
+If your application registration is multi-tenant, leave this parameter blank.
+For more information, see link:https://learn.microsoft.com/en-us/azure/active-directory/manage-apps/home-realm-discovery-policy[Home Realm Discovery].
++
+.`{my-app-config-file}` file fragment with optional `domainHint` field
+[source,yaml,subs="+quotes,+attributes"]
+----
+auth:
+  environment: production
+  providers:
+    microsoft:
+      production:
+        domainHint: ${AUTH_AZURE_TENANT_ID}
+----
+
+`additionalScopes`::
+Optional for additional scopes.
+To add scopes for the application registration, uncomment and enter the list of scopes that you want to add.
+The default and mandatory value lists: `'openid', 'offline_access', 'profile', 'email', 'User.Read'`.
++
+.`{my-app-config-file}` file fragment with optional `additionalScopes` field
+[source,yaml,subs="+quotes,+attributes"]
+----
+auth:
+  environment: production
+  providers:
+    microsoft:
+      production:
+        additionalScopes:
+           - Mail.Send
+----
+
+`sessionDuration`::
+Lifespan of the user session.
+Enter a duration in `ms` library (such as '24h', '2 days'), ISO duration, or "human duration" format.
++
+.`app-config-rhdh.yaml` fragment with optional `sessionDuration` field
+[source,yaml,subs="+quotes"]
+----
+auth:
+  providers:
+    microsoft:
+      production:
+        sessionDuration:
+          hours: 24
+----
+
+`signIn`::
+
+`resolvers`:::
+After successful authentication, {product-short} resolves the user signing in to an existing user in the {product-short} catalog.
+To best match users securely for your use case, consider configuring a specific resolver.
++
+Enter the resolver list to override the default resolver: `userIdMatchingUserEntityAnnotation`.
++
+The authentication provider tries each sign-in resolver in order until it succeeds, and fails if none succeed.
++
+WARNING: In production mode, only configure one resolver to ensure users are securely matched.
++
+.`app-config-rhdh.yaml` fragment with optional field to allow signing in users absent from the software catalog
+[source,yaml]
+----
+auth:
+  environment: production
+  providers:
+    microsoft:
+      production:
+        clientId: ${AUTH_AZURE_CLIENT_ID}
+        clientSecret: ${AUTH_AZURE_CLIENT_SECRET}
+        tenantId: ${AUTH_AZURE_TENANT_ID}
+        signIn:
+          resolvers:
+            - resolver: usernameMatchingUserEntityName
+              dangerouslyAllowSignInWithoutUserInCatalog: true
+signInPage: microsoft
+----
+
+`resolver`::::
+Enter the sign-in resolver name.
+Available resolvers:
+
+* `userIdMatchingUserEntityAnnotation`
```

Comment:
```suggestion
* `emailMatchingUserEntityAnnotation`
```

This resolver: "Looks up the user by matching their Microsoft email to the email entity annotation."